EIPS/eip-2159.md

```diff
@@ -3,7 +3,7 @@ eip: 2159
 title: Common Prometheus Metrics Names for Clients
 author: Adrian Sutton (@ajsutton)
 discussions-to: https://ethereum-magicians.org/t/common-chain-metrics/3415
-status: Draft
+status: Last Call
```

Comment:
This status needs a `review-period-end` field.

Reply:
Thank you, I knew I was missing something...